File: example/launch-darkly/js/sample-LD.ts
FeatureFlag.start("your_env_id", "your_api_key");

const visitor = FeatureFlag.newVisitor({
    visitorId: "your_visitor_id",
    context: { isVip: true },
});

visitor.on("ready",  (error) => {
    if (error) {
        return;
    }

    const btnColorFlag: string = client.variation("LD-string-flag-ts", 'red');
    const backgroundSize: number = client.variation("LD-number-flag-ts", 16.3);
    const showBackground: boolean = client.variation("LD-bool-flag-ts", true);

    const btnColorFlag: string = client.variationDetail("LD-string-flag-ts-1", 'green');
    const backgroundSize: number = client.variationDetail("LD-number-flag-ts-1", 16);
    const showBackground: boolean = client.variationDetail("LD-bool-flag-ts-1", false);

});

visitor.fetchFlags();

File: example/launch-darkly/java/sample-LD.java
Visitor visitor = featureFlag.newVisitor("visitor_unique_id").build();
visitor.updateContext("isVip", true);
visitor.fetchFlags().whenComplete((instance, error) -> {
    
    Flag<String> btnColorFlag = visitor.stringVariation("LD-string-flag-java", context, "red");
    Flag<Number> backgroundColorFlag = visitor.intVariation("LD-number-flag-java", context, 13);
    Flag<Boolean> backgroundColorFlag = visitor.boolVariation("LD-bool-flag-java", context, true);

    Flag<String> btnColorFlag = visitor.stringVariationDetail("LD-string-flag-java-1", context, "red");
    Flag<Number> backgroundColorFlag = visitor.intVariationDetail("LD-number-flag-java-1", context, 13);
    Flag<Boolean> backgroundColorFlag = visitor.boolVariationDetail("LD-bool-flag-java-1", context, true);

});

